### ISSUE TITLE
Four leading spaces before descriptions?

### ISSUE THREAD

**Issue by kennethgillen** [blangel/wrk#9]
Hi,

Using wrk version 1.0.1, there appears to be four leading spaces printed in front of a card description when using `wrk desc`

Since these spaces are not actually in the card description it's frustrating to have them included when running e.g. `wrk desc` - see the 'last name' example below:

HOSTNAME:~ kenny$ wrk desc c:134561234132412341
Description of card 134561234132412341:
  card-title | 134561234132412341
    first name: 'forename'
last name: 'surname'
institution: 'some-institution'
email: 'some-email@address.com'
    https://trello.com/c/aFaFaFaF/card-title-here

I wonder if it's the following lines:

`HOSTNAME:wrk kenny$ git grep -i "    %s" *
src/main/java/net/ocheyedan/wrk/cmd/trello/Desc.java:                    Output.print("    %s", desc);
src/main/java/net/ocheyedan/wrk/cmd/trello/Desc.java:                    Output.print("    %s", desc);
src/main/java/net/ocheyedan/wrk/cmd/trello/Desc.java:                    Output.print("    %s", desc);`

**Comment by blangel** [comment]
@kennethgillen right. The spaces are for indentation but this fails when the description itself has newlines within it. Your output (as I intended) should have been:
```
Description of card 134561234132412341:
  card-title | 134561234132412341
    first name: 'forename'
    last name: 'surname'
    institution: 'some-institution'
    email: 'some-email@address.com'
    https://trello.com/c/aFaFaFaF/card-title-here
```
Would this suffice? Alternatively, I could simply not indent everything:
```
Description of card 134561234132412341:
card-title | 134561234132412341
first name: 'forename'
last name: 'surname'
institution: 'some-institution'
email: 'some-email@address.com'
https://trello.com/c/aFaFaFaF/card-title-here
```

**Comment by kennethgillen** [comment]
Hi @blangel.

I think the former, i.e. 

```
Description of card 134561234132412341:
  card-title | 134561234132412341
    first name: 'forename'
    last name: 'surname'
    institution: 'some-institution'
    email: 'some-email@address.com'
    https://trello.com/c/aFaFaFaF/card-title-here
```

Is lovely, and looks great. Consistency looks good. Thanks for noticing the issue.
